src/test/groovy/org/prebid/server/functional/tests/privacy/CcpaSpec.groovy

```diff
@@ -0,0 +1,199 @@
+package org.prebid.server.functional.tests.privacy
+
+import org.prebid.server.functional.model.bidder.BidderName
+import org.prebid.server.functional.model.config.AccountCcpaConfig
+import org.prebid.server.functional.model.config.AccountConfig
+import org.prebid.server.functional.model.config.AccountPrivacyConfig
+import org.prebid.server.functional.model.db.Account
+import org.prebid.server.functional.model.db.StoredRequest
+import org.prebid.server.functional.testcontainers.PBSTest
+import org.prebid.server.functional.util.privacy.BogusConsent
+import org.prebid.server.functional.util.privacy.CcpaConsent
+import spock.lang.PendingFeature
+
+import static org.prebid.server.functional.util.privacy.CcpaConsent.Signal.ENFORCED
+
+@PBSTest
+class CcpaSpec extends PrivacyBaseSpec {
+
+    // TODO: extend ccpa test with actual fields that we should mask
+    def "PBS should mask publisher info when privacy.ccpa.enabled = true in account config"() {
+        given: "Default ccpa BidRequest"
+        def validCcpa = new CcpaConsent(explicitNotice: ENFORCED, optOutSale: ENFORCED)
+        def bidRequest = getCcpaBidRequest(validCcpa)
+
+        and: "Save account config into DB"
+        def ccpa = new AccountCcpaConfig(enabled: true)
+        def privacy = new AccountPrivacyConfig(ccpa: ccpa)
+        def accountConfig = new AccountConfig(privacy: privacy)
+        def account = new Account(uuid: bidRequest.site.publisher.id, config: accountConfig)
+        accountDao.save(account)
+
+        when: "PBS processes auction request"
+        defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Bidder request should contain masked values"
+        def bidderRequests = bidder.getBidderRequest(bidRequest.id)
+        def maskedGeo = getMaskedGeo(bidderRequests)
+        assert bidderRequests.device?.geo?.lat == maskedGeo["lat"]
```

Comment:
You're comparing bidder request to bidder request here. This is incorrect. Please provide your own device.geo and compare against that. Same for other tests.